pkg/endpoint/endpoint.go

```diff
@@ -438,6 +438,18 @@ func (e *Endpoint) GetLabels() []string {
 	return e.SecLabel.Labels.GetModel()
 }
 
+
+// GetLabels returns the SHA of labels
+func (e *Endpoint) GetLabelSHA() string {
+	if e.SecLabel == nil {
+		return ""
+	}
+	if e.SecLabel.LabelSHA256 == "" {
+		e.SecLabel.LabelSHA256 = e.SecLabel.Labels.SHA256Sum()
```

Comment:
Given that we are not updating `LabelsSHA256` right now every time we update the labels, can we simply always calculate the hash here for now? We can cache the hash once we are sure that we update the cache reliably.